### DISCUSSION TITLE
I need help in starting open source works

### DISCUSSION THREAD

**Discussion by josephhh47** [nginxinc/nginx-plus-go-client#242] [Q&A] [answered]
Hello everyone! I'm new to coding and I want to start contributing to the open source works how and where should I start?

Thank you in advance!

**Comment by probablynotnmp** [comment, marked as answer]
Hey Joseph! Here's are some things that'll help you start in open source:

1) Learn Git and GitHub: Familiarize yourself with version control and the GitHub platform.
2) Explore Beginner Projects: Find beginner-friendly projects with "good first issue" labels.
3) Contribute Documentation/Bug Fixes: Start with tasks like documentation updates or bug fixes.
4) Join Communities: Participate in coding and open-source communities for guidance.
5) Read Contributing Guidelines: Follow project guidelines for submitting contributions.
6) Start Small: Begin with manageable tasks to build confidence.
7) Seek Feedback: Ask for feedback to improve your skills.
8) Persist and Learn: Stay persistent, keep learning, and don't be afraid to ask questions.

If you have any doubts feel free to reach out to me ;)
